setup.cfg

```diff
@@ -1,3 +1,15 @@
+[isort]
+skip =
+    _vendor
+    __main__.py
+multi_line_output = 5
+known_third_party =
+    pip._vendor
+known_first_party =
+    pip
+    tests
+default_section = THIRDPARTY
```

Comment:
Any package that is not from `pip` or `tests` would be treated as a third-party import.